packages/logos-docusaurus-theme/src/client/components/mdx/TimelineItem/TimelineItem.scss

```diff
@@ -3,14 +3,18 @@
 
 .mdx-timeline-item {
   width: 236px;
-  height: 100%;
+  height: 306px;
```

Comment:
Noticed the implementation breaks the optional `alignment` property. I'd suggest removing the property.

Reply:
The top-aligned variant is still not working. I would suggest removing the `alignment` property and the top-aligned styles since it's not implemented on Figma anyway